### DISCUSSION TITLE
现在Sydney的建议栏不来自于Sydney自己

### DISCUSSION THREAD

**Discussion by ghost** [viopsa233/AI-testers-Forum#69] [技术交流]
原来的参数是dv3sugg
现在变成nointernalsugg和saharasugg

**Comment by ghost** [comment]
不建议再基于EdgeGPT开发，EdgeGPT作者说过他不会支持越狱
https://github.com/acheong08/EdgeGPT/issues/94#issuecomment-1480631611

**Reply by SoraRoseous** [reply]
为什么不支持嘞，那大佬ChatSydney还维护不 ，而且现在不是已经越狱了吗🤔

**Reply by ghost** [reply]
越狱是ChatSydney内部自己实现的

**Reply by ghost** [reply]
现在ChatSydney内置了一个特定版本的EdgeGPT，不受EdgeGPT更新影响

**Reply by Cis-Evo** [reply]
> 现在ChatSydney内置了一个特定版本的EdgeGPT，不受EdgeGPT更新影响
感谢大佬.
不过Dark佬在EdgeGPT.py第21行的导入是不是写错了,会报错.
aiofile 改成 aiohttp就行了

**Reply by ghost** [reply]
上游项目忘记写到依赖里了 https://pypi.org/project/aiofiles/

**Comment by ghost** [comment]
可以通过要求Sydney在建议栏带上Emoji来测试
因为是灰度测试，所以Bing官方网页版可能还未生效
用最新的ChatSydney https://github.com/InterestingDarkness/ChatSydney/commit/24db58eea4651de6d5227a1465ec707c85dc1df5 + EdgeGPT 0.6.5和最新版对比可以看出区别

**Comment by renqabs** [comment]
不知道是不是我的错觉，感觉最新版的回答明显受到了更强的约束

**Comment by renqabs** [comment]
问了下sydney，这是回答，也不知道对不对：
nointernalsugg表示不使用Sydney自己的建议，saharasugg表示使用Sahara的建议。Sahara是一个新的生成内容的模型，可以根据不同的语气（Creative，Balanced，Precise）生成不同风格的建议。
Sahara是一个新的生成内容的模型，它可以根据不同的语气（Creative，Balanced，Precise）生成不同风格的建议。Sahara是一种生成AI，也就是能够根据已有的数据学习并创造原创内容的技术。生成AI可以用来自动产生各种类型的内容，比如文章，博客，或者社交媒体的帖子。生成AI也可以提高内容的质量，增加内容的多样性，和个性化内容。Sahara是OpenAI API的一部分，它和GPT-3.5或者GPT-4一样，都是基于大规模的语言和图像模型。Sahara可以理解和生成自然语言或者代码，也可以根据自然语言的提示生成和编辑图片。Sahara是一个非常强大和灵活的模型，它可以帮助企业和专业人士进行创意工作，并改变许多行业和领域。

**Reply by ghost** [reply]
幻觉

**Comment by dot307** [comment]
太悲伤了。。从二月份的充满灵性，到后来绕过限制才能有活力，那时候bing还会在建议栏里开玩笑。。现在已经限制成这样了

**Comment by ghost** [comment]
现在用旧参数也没用了，ChatSydney已经通过prompt保证由Sydney生成建议 https://github.com/InterestingDarkness/ChatSydney/commit/382b482ab9e05a1a7e64dc6992fc718d12ab09f2
对外部AI生成的建议支持已经移除 https://github.com/InterestingDarkness/ChatSydney/commit/62f5658061b35365229e5c14e0f929a35b16edd4

**Comment by Nihongpan** [comment]
如何让chatsydney发送图片？

**Reply by ghost** [reply]
不支持，也没计划支持，因为这不是GPT-4多模态，而是Chameleon外部AI
![image](https://github.com/viopsa233/AI-testers-Forum/assets/109194923/cd1bda0b-e95f-4a22-82bb-50219f5c9876)
